src/common/Gateway.sol

```diff
@@ -44,7 +44,7 @@ contract Gateway is Auth, Recoverable, IGateway {
     mapping(PoolId => mapping(address => bool)) public manager;
 
     // Outbound & payments
-    address public transient batcher;
+    address transient _batcher;
```

Comment:
Why is this _batcher instead of batcher now?

Reply:
Because batcher is private, to ensure it's correctly read, you need to go through `lockCallback()`

Reply:
I made internal, in case in the future we want to improve test coverage